src/amo/components/Home/index.js

```diff
@@ -220,6 +221,7 @@ export class HomeBase extends React.Component {
         />
 
         <LandingAddonsCard
+          addonInstallSource="mostpopular"
```

Comment:
I wonder if these strings should be constants?

Reply:
What would that buy us? I could do it but I'm not sure how helpful it would be.

Reply:
Aren't they used in multiple places?

Reply:
No, each one is only configured in a single place. I suppose in the future we might put some in multiple places.

Reply:
If it's not used in multiple places it's less useful, but if it ever was it might be nice to validate these are one of a list of valid sources.